pcc/src/cli/index.ts

```diff
@@ -28,13 +28,67 @@ yargs(hideBin(process.argv))
           type: 'string',
           choices: ['nextjs', 'gatsby'],
           demandOption: true,
+        })
+        .option('appName', {
+          describe: '"package.json" compatible name for the project.',
+          type: 'string',
+          demandOption: false,
+        })
+        .option('noInstall', {
+          describe: 'Do not install any dependencies.',
+          type: 'boolean',
+          default: false,
+          demandOption: false,
+        })
+        .option('use-npm', {
+          describe: 'Use NPM package manager for installing dependencies.',
+          type: 'boolean',
+          default: false,
+          demandOption: false,
+        })
+        .option('use-pnpm', {
+          describe: 'Use PNPM package manager for installing dependencies.',
+          type: 'boolean',
+          default: false,
+          demandOption: false,
+        })
+        .option('use-yarn', {
+          describe: 'Use Yarn package manager for installing dependencies.',
+          type: 'boolean',
+          default: true,
+          demandOption: false,
+        })
+        .option('silent', {
```

Comment:
@kevinstubbs do we need to add `silent` options to other commands as well? Currently I have added it only for `pcc init`

Reply:
I'm alright with having it for just the init command for now. We can add to others later if a feature request comes in.